test/e2e/e2e.go

```diff
@@ -815,7 +819,16 @@ func deleteMetalLB(metallb *metallbv1beta1.MetalLB) {
 	}, timeout, interval).Should(BeTrue())
 }
 
-var _ = BeforeSuite(func() {
-	_, err := testclient.Client.Namespaces().Get(context.Background(), OperatorNameSpace, metav1.GetOptions{})
-	Expect(err).ToNot(HaveOccurred(), "Should have the MetalLB Operator namespace")
-})
+func getMetalLB() (*metallbv1beta1.MetalLB, error) {
+	metallb := &metallbv1beta1.MetalLB{}
+	if UseMetallbResourcesFromFile {
```

Comment:
Can we unroll and avoid the else / extra nesting by returning?

Something like 
```
if !UseMetallbResources {
    return &metallbv1beta1.MetalLB{Name: xx, Namespace : yy}, nil
}

... Load from file and return

Reply:
fixed